.taskcluster.yml

```diff
@@ -194,11 +194,6 @@ tasks:
                           && cd reference-browser
                           && git checkout ${event.release.tag_name}
                           && python automation/taskcluster/decision_task_nightly.py \
-                            --commit \
```

Comment:
Okay to get rid of that option too

Reply:
I use this option in my staging hook :)

Reply:
Oh, err: are you referring to `--date` or `--staging`? :thinking:

Reply:
`--commit` which I'm sure you willingly removed 😃 https://github.com/mozilla-mobile/reference-browser/pull/687/files#diff-59f90d509bc6ae2c9d877250c5ba49c5L91